Content.Shared/Lock/Lockpick/LockpickComponent.cs

```diff
@@ -0,0 +1,18 @@
+using Robust.Shared.Audio;
+namespace Content.Shared.Lock.Lockpick;
+
+[RegisterComponent]
+public sealed partial class LockpickComponent : Component
+{
+    //Plays when starting the lockpicking
```

Comment:
Put the comments in the `</summary>` tags! If you do it like that it'll display the comments when you hover over the variable. You can find plenty of examples but here is one:
```
/// <summary>
///     Description here!
///     Description here!
/// </summary>
```